tests/test_freshdesk_all_fields.py

```diff
@@ -0,0 +1,101 @@
+from tap_tester import runner, connections, menagerie
+
+from base import FreshdeskBaseTest
+
+# As we are not able to generate following fields by Freshdesk UI, so removed it from expectation list.
+KNOWN_MISSING_FIELDS = {
+    'tickets': {
+        'facebook_id',
+        'description',
+        'description_text',
+        'twitter_id',
+        'name',
+        'phone',
+        'email'
+    },
+    'groups': {
+        'auto_ticket_assign',
+        'agent_ids'
+    },
+    'agents': {
+        'group_ids',
+        'role_ids'
+    },
+    'contacts': {
+        'view_all_tickets',
+        'other_companies',
+        'other_emails',
+        'tags',
+        'avatar'
+    }
+}
+
+class TestFreshdeskAllFields(FreshdeskBaseTest):
+    """Test that with all fields selected for a stream automatic and available fields are replicated"""
+
+    @staticmethod
+    def name():
+        return "tap_tester_freshdesk_all_fields"
+
+    def test_run(self):
+        """
+        • Verify no unexpected streams were replicated
+        • Verify that more than just the automatic fields are replicated for each stream. 
+        • Verify all fields for each stream are replicated
+        """
+        
+        # To collect "time_entries", "satisfaction_ratings" pro account is needed. Skipping them for now.
+        expected_streams = self.expected_streams() - {"time_entries", "satisfaction_ratings"}
```

Comment:
Seems like we are removing these streams in all tests so can we remove this in base.py and create a separate set of it and add that in the discovery test?

Reply:
Updated expected_streams function in base.py